tests/components/alexa/test_smart_home.py

```diff
@@ -693,6 +693,134 @@ def test_unknown_sensor(hass):
     yield from discovery_test(device, hass, expected_endpoints=0)
 
 
+@asyncio.coroutine
```

Comment:
We've migrated to `async`/`await` syntax. Please drop this decorator and prefix async methods with `async` like `async def test_thermostat`. Replace any `yield from` with `await`.